proposals/csharp-7.0/tuples.md

```diff
@@ -290,37 +311,37 @@ System.ValueTuple<int, int> vt = t;  // identity conversion
 
 t2.moo = 1;
 ```
+end example\]
 
-> note:
->
-> An element name at one position on one side of a conversion, and the same name at another position on the other side almost certainly have bug in the code:
+In teh case in which an element name at one position on one side of a conversion, and the same name at a different position on the other side, the copmpiler shall issue a warning.
 
+\[Example:
 ```csharp
 (string first, string last) GetNames() { ... }
 (string last, string first) names = GetNames(); // Oops!
 ```
-
-> Compilers should issue a warning for the preceding code.
->
-> endnote.
+end example\]
 
 ### Boxing conversions
 
-> The following text should be added to [Boxing conversions](../../spec/conversions.md#boxing-conversions) after the first paragraph:
+> Add the following text to [Boxing conversions](../../spec/conversions.md#boxing-conversions) after the first paragraph:
 
-Tuples, like all value types, have a boxing conversion. Importantly, the names aren't part of the runtime representation of tuples, but are tracked only by the compiler. Thus, once you've "cast away" the names, you cannot recover them. In alignment with the identity conversions, a boxed tuple will unbox to any tuple type that has the same element types in the same order.
+Tuples have a boxing conversion. Importantly, the element names aren't part of the runtime representation of tuples, but are tracked only by the compiler. Thus, once element names have been "cast away", they cannot be recovered. In alignment with identity conversion, a boxed tuple unboxes to any tuple type that has the same element types in the same order.
 
 ### Tuple conversions
 
-> This section should be added after [Implicit enumeration conversions](../../spec/conversions.md#implicit-enumeration-conversions)
+> Add this section after [Implicit enumeration conversions](../../spec/conversions.md#implicit-enumeration-conversions)
 
-Tuple types and expressions support a variety of conversions by "lifting" conversions of the elements into overall *tuple conversion*.
-For the classification purpose, all element conversions are considered recursively. For example: To have an implicit conversion, all element expressions/types must have implicit conversions to the corresponding element types.
+Tuple types and expressions support a variety of conversions by "lifting" conversions of the elements into overall *tuple conversion*. For the classification purpose, all element conversions are considered recursively. For example, to have an implicit conversion, all element expressions/types shall have implicit conversions to the corresponding element types.
 
 Tuple conversions are *Standard Conversions* and therefore can stack with user-defined operators to form user-defined conversions.
 
+**ISSUE:** Is 'stack' a defined term in this context?
```

Comment:
I don't think we've defined it. I'd prefer using different language in the previous sentence to introducing new terms. What are your thoughts?

Reply:
How about the following tweak?

>Tuple conversions are *Standard Conversions* and therefore can **combine** with user-defined operators to form user-defined conversions.

Reply:
I think "be combined" would be best.